src/Tabs/Tabs.tsx

```diff
@@ -109,7 +109,7 @@ class Tabs extends React.Component<TabsProps, TabsState> {
     const spaceProps = getSubset(this.props, propTypes.space);
     return (
       <Box position="relative">
-        <FocusManager refs={this.tabRefs}>
+        <FocusManager refs={this.tabRefs} selectedIndex={this.getSelectedIndex()}>
```

Comment:
At a second look I think this prop name looks strange. Should we rather call this `defaultFocusedIndex` instead? I just think that the focus manager shouldn't care about the selected value's index.

Reply:
I agree, this is a much better name!